core/domain/collection_domain_test.py

```diff
@@ -641,9 +619,33 @@ class SchemaMigrationUnitTests(test_utils.GenericTestBase):
 tags: []
 title: A title
 """)
+    YAML_CONTENT_V3 = ("""category: A category
+language_code: en
+nodes:
+- acquired_skills:
+  - 0
+  - 1
+  exploration_id: Exp1
+  prerequisite_skills: []
+objective: ''
+schema_version: 2
+skills:
+  0:
+    skill_id: 0
```

Comment:
going back to this, I'm not entirely sure if skills need an id field at all :/ It seems like every time I'm accessing skills, I'm doing so through the dict anyway and know the ID. What do you think?